package.json

```diff
@@ -328,7 +328,7 @@
     "jest-canvas-mock": "^2.3.1",
     "jsdom": "^11.2.0",
     "koa": "^2.7.0",
-    "lavamoat": "^5.3.5",
+    "lavamoat": "^6.1.1",
```

Comment:
What new changes are in this release? Does this now rely on `@lavamoat/aa`? Are there other improvements we should know about? If you have a secret changelog then I'd love to see it :)

Reply:
some highlights
1. policy's use package paths
2. policy override is only used when generated the policy, so the policy file is exactly what gets used
3. packages added in the override are walked automatically for policy (though this has been observed not working in this metamask branch)

Reply:
Cool, thanks!